package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "astro-velo",
-  "version": "0.7.0",
+  "version": "0.7.1",
   "author": "Mobify",
   "dependencies": {
     "astro-sdk": "^0.7.0",
```

Comment:
Don't need to anymore.  The `^` means that npm will use the most up-to-date `0.7.*` release of Astro.